### PR TITLE
SV Data Model v2

### PR DESCRIPTION
**Pull request by drio18**
This PR updates the SV data model for the v2 pipeline, bringing in new fields at both the variant and sample level. A new calcprop is added that provides a summary of gene characteristics to be shown in the annotation space.

Additionally, a fix is made to the shared SNV/SV ingestion code for boolean fields such that true and false values are explicitly defined and no default is used. This fixes the minor bug occurring for `transcript.csq_strand` for both variant types.

Tests were updated accordingly for the new fields as well as for the ingestion fix, and the v2 SV VCF as well as an SNV VCF were ingested with this branch without errors.

### CODE REVIEW

**Review by phil-grayson — APPROVED** [review]
Thanks Doug - I see the new fields I expect.

File: src/encoded/types/structural_variant.py [hunk 1]
```diff
@@ -160,6 +160,87 @@ def size_display(self, request, START, END):
...
+        contained_count = 0
+        breakpoint_count = 0
+        omim_count = 0
+        if transcript:
+            for item in transcript:
+                gene = item.get("csq_gene")
+                if gene and gene not in genes:
+                    genes.append(gene)
+        for gene in genes:
+            gene = get_item_or_none(request, gene, "Gene", frame="raw")
```
Comment (willronchetti) [inline src/encoded/types/structural_variant.py:224]: How many genes do we expect here? Note that you are bounding the time to render the object/embedded view by the number of associated genes.

**Review by willronchetti — APPROVED** [review]

File: src/encoded/ingestion/vcf_utils.py [hunk 2]
```diff
@@ -99,6 +99,8 @@ class VCFParser(object):
     GT_REF = '0/0'
     GT_REF_PHASED = '0|0'
     GT_MISSING = './.'
+    BOOLEAN_TRUE = ["1", "YES", True]
+    BOOLEAN_FALSE = ["0", "-1", ""]
```
[the remark this answers is not in the document]
Comment (drio18) [reply src/encoded/ingestion/vcf_utils.py:103]: I'll add it in as something to catch, but it's currently not needed due to how VEP annotates variants. The only fields set to `True` are those that are flags added by VEP, and such flags are only added when the variant meets the criteria, so there's never a `False` value for the field.

File: src/encoded/schemas/structural_variant_sample.json [hunk 3]
```diff
@@ -58,6 +58,17 @@
             "scope": "sample_variant",
             "schema_title": "Sample ID"
         },
+        "GQ": {
```
Comment (sbreiff) [inline src/encoded/schemas/structural_variant_sample.json:61]: Do we want to add this as a facet? That's fine if not, but it's in the SNV facets so I wanted to check.

**Review by sbreiff — APPROVED** [review]
Looks good to me.

**Review by Bianca-Morris — APPROVED** [review]
I think it also looks good for me. Just need to switch around `size_display` and `size` cols (not sure if you wanted to include that in this PR or not)

**Pull request merged**